### DISCUSSION TITLE
Donation requests????????????

### DISCUSSION THREAD

**Discussion by vietnamveteran** [smplayer-dev/smplayer#1069] [General]
someone keeps popping up while i am using smplayer  asking for  a donation
it doe not give a way to reply i have already given a donation
every where on the internet it states that it is a FREE PLAYER 
every time i turned i turned around here came another request and frankly i am getting tired of it and i finally gave you a one time donation of $40.00  but guess what the popups kept coming
i feel that is a fair amount of money to use your player and i really like it
but on the popups you should add a  line that says i  have already donated  
thank you for your time and understanding
you make a great player
Rick

**Comment by smplayer-dev** [comment]
The popup window should only appear twice. Additionally, once you click the "Donate" button, the popup will no longer appear, even if you choose not to donate.

If this isn't happening, there may be an issue with the configuration folder—SMPlayer might be unable to save the settings. Another possibility is that you selected the option to delete the previous configuration when installing a new version.
